spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,42 @@
+feature 'View Peeps' do
+  scenario 'list of peeps on homepage' do
+    # Dummy user
+    user = user_create_dave
+
```

Comment:
maybe you could call variable **dumy_user** just to make it more clear and then you wouldn't need the comment